README.md

```diff
@@ -139,6 +139,12 @@ You can deploy this project using any host of your choice
 - GitHub: [@Bigizi Nduwayo Crispin](https://github.com/Bigizi)
 - LinkedIn: [@Bigizi Nduwayo Crispin](https://www.linkedin.com/in/bigizi/)
 
```

Comment:
- [OPTIONAL ] I was not able to inline where the change should be made but I encourage you to remove microverse logo on your readme and make your own logo or remove it from your readme :wink: , I can see that all your branches don't have it , kindly remove it since this is your own project :wink: 
![Screenshot from 2023-04-25 00-10-49](https://user-images.githubusercontent.com/88894525/234128149-d3c0df3b-19e7-4449-b2ab-a75668818794.png)